ui-app/client/src/components/FormStorageEditor/editors/OptionTypeEditor.tsx

```diff
@@ -0,0 +1,304 @@
+import React, { ChangeEvent, Fragment, useEffect, useState } from 'react';
+import {
+	CustomSchema,
+	FormCompDefinition,
+	FormCompValidation,
+	Option,
+	compValidationMap,
+	CompValidations,
+} from '../components/formCommons';
+import { deepEqual, duplicate } from '@fincity/kirun-js';
+import CommonTextBox from '../components/CommonTextBox';
+import CommonCheckBox from '../components/CommonCheckBox';
+
+const OptionComp = ({
```

Comment:
follow the function OptionComp(props: {}) {} format don't use const function for React Components.